lte/gateway/python/magma/pipelined/app/arp.py

```diff
@@ -13,13 +13,13 @@
 
 from collections import namedtuple
 
+import getmac
```

Comment:
Unfortunately, this messes with mocks used elsewhere since python mocks based on names. In related test code I would have to write something like
```
@patch("magma.pipelined.app.arp.get_mac_address", return_value=...)
```
Instead of the perhaps more obvious
```
@patch("getmac.get_mac_address", return_value=...)
```